### PR TITLE
Feature/omvarldskunskap

### PR DESCRIPTION
**Pull request by docNord**
Wrote an article about the study group on the local/global politics/economics suggested by Hans Abrahamsson at the last AW of 2023. Also added an event for the next meeting of the study group in case someone is interested.

Please read through and provide feedback on the text, it probably needs some work (and possibly a picture).

### CODE REVIEW

File: content/posts/event-omvarldskunskap1-2024-01-27.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+title: Omvärldskunskap del I - samtal om kursen
```
Comment (mgcth) [inline content/posts/event-omvarldskunskap1-2024-01-27.md:1]: Could the file be named `-del1-` maybe?

File: content/posts/event-omvarldskunskap1-2024-01-27.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+date: 2024-01-27
+modified:
+author: Anders Nord
+category: Event
+tags: politik, vetenskap, klimat, aw, social hållbarhet, demokrati
+status: published
+
+Efter IfMs <a href="event-aw-2023-12-06.md">AW</a> med Hans Abrahamsson har flera
```
Comment (mgcth) [inline content/posts/event-omvarldskunskap1-2024-01-27.md:9]:
```suggestion
Efter IfMs [AW](event-aw-2023-12-06.md) med Hans Abrahamsson har flera
```

However, can we link md files like that internally or should it be

```[AW](/posts/event-aw-2023-12-06)```

?

File: content/posts/event-omvarldskunskap1-2024-01-27.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+i föreningen valt att i studiecirkelformat läsa in självstudiematerialet om "konsten
+att värna demokrati och social hållbarhet" som Hans har tagit fram i en större
+forskningsstudie för Sveriges Kommuner och Regioner (SKR). Materialet finns tillgängligt
+för gratis nedladdning på en speciell hemsida som de tagit fram: <a href="www.omvarldskunskap.se"
+target="_blank">www.omvarldskunskap.se</a>
```
Comment (mgcth) [inline content/posts/event-omvarldskunskap1-2024-01-27.md:14]:
```suggestion
target="_blank">www.omvarldskunskap.se</a>.
```

File: content/posts/studiecirkel-omvarldskunskap2024-01-23.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+Ingenjörsarbete För Klimatets medlemmar närvarade, vilket inspirerat en studiecirkel
+för att läsa in Hans kurs <a href="https://omvarldskunskap.se">omvärldskunskap.se</a>.**
+
+Kursen togs ursprungligen fram i samband med Sveriges Kommuner och Regioner (SKR),
+och syftar till att introducera lokala politiska ledare till den globala ekonomiska
+och politiska spelplanen och dess föränderliga förutsättningar. Den består av ett
+gediget kursmaterial i form av text och filmade föreläsningar uppdelat i tre delar.
+
+Studiecirkeln träffas Lördagen den 27/1 för att diskutera del I av kursen.
+<a href="event-omvarldskunskap1-2024-01-27">Event</a>
```
Comment (mgcth) [inline content/posts/studiecirkel-omvarldskunskap2024-01-23.md:21]: Is this part of the above sentence?

File: content/posts/studiecirkel-omvarldskunskap2024-01-23.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+title: Studiecirkel omvärldskunskap
```
Comment (mgcth) [inline content/posts/studiecirkel-omvarldskunskap2024-01-23.md:1]: Could we add a hyphen to this filename to for consistency?

File: content/posts/event-omvarldskunskap1-2024-01-27.md (hunk 1 above)
Comment (docNord) [reply content/posts/event-omvarldskunskap1-2024-01-27.md]: Fixed.

File: content/posts/studiecirkel-omvarldskunskap2024-01-23.md (hunk 5 above)
Comment (docNord) [reply content/posts/studiecirkel-omvarldskunskap2024-01-23.md:1]: Fixed.

**Review by mgcth — APPROVED** [review]
Looks good to me!

**Pull request merged**